modules/Core/includes/endpoints/ServerInfoEndpoint.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+/**
+ * @param array $info Minecraft server info
+ *
+ * @return string JSON Array
+ */
+class ServerInfoEndpoint extends EndpointBase {
+
+    public function __construct() {
+        $this->_route = 'serverInfo';
+        $this->_module = 'Core';
+        $this->_description = 'Update the Minecraft server information NamelessMC tracks';
+    }
+
+    public function execute(Nameless2API $api) {
```

Comment:
Another type

Reply:
As per our conversation on Discord, leaving this.